cli/azd/pkg/project/scaffold_gen_test.go

```diff
@@ -18,23 +18,23 @@ func Test_genBicepParamsFromEnvSubst(t *testing.T) {
 		want       string
 		wantParams []scaffold.Parameter
 	}{
-		{"foo", false, "'foo'", nil},
-		{"${MY_VAR}", false, "myVar", []scaffold.Parameter{{Name: "myVar", Value: "${MY_VAR}", Type: "string"}}},
+		{"foo", false, "foo", nil},
+		{"${MY_VAR}", false, "${myVar}", []scaffold.Parameter{{Name: "myVar", Value: "${MY_VAR}", Type: "string"}}},
```

Comment:
As commented above:
1. For text, remove quote.
2. For variable, add `${` and `}`.